app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginFragment.java

```diff
@@ -64,6 +63,8 @@ public void onStart() {
         loginFragmentViewModel.getProgress().observe(this, this::showProgress);
         loginFragmentViewModel.getError().observe(this, this::showError);
         loginFragmentViewModel.getEmailList().observe(this, this::attachEmails);
+        loginFragmentViewModel.getDecryptedEmail().observe(this, email -> binding.emailDropdown.setText(email));
+        loginFragmentViewModel.getDecryptedPassword().observe(this, password -> binding.userPassword.setText(password));
```

Comment:
Even two extra endpoints are added. We need to decrease calls in views, not increase

Reply:
I haven't removed them  because the edit text fields aren't getting set if I am setting them via xml. I have tested it. I can upload the gif as well.

Reply:
Did you push the loaded encrypted value through the LiveData, because I haven't seen that

Ideally, you should be pushing the login and binding it. The two way binding will also work then

Reply:
Why would I need to push the encrypted value through LiveData. I just want to observe the decrypted string in the view